src/behaviors/StopBehavior.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\queue\behaviors;
+
+use yii\base\Behavior;
+use yii\caching\Cache;
+use yii\di\Instance;
+use yii\queue\ExecEvent;
+use yii\queue\Queue;
+
+/**
+ * Stoppable behavior allow stopping scheduled jobs in a queue.
+ *
+ * This behavior provides a [[stop()]] method, which allows to mark scheduled jobs as "stopped", which
+ * will prevent their execution.
+ *
+ * This behavior should be attached to the [[Queue]] component.
+ *
+ * @author Roman Zhuravlev <zhuravljov@gmail.com>
+ * @since 2.0.1
+ */
+class StopBehavior extends Behavior
+{
+    /**
+     * @var Cache|array|string the cache instance used to store stopped status.
+     */
+    public $cache = 'cache';
+    /**
+     * @var bool
```

Comment:
The option allows to turn off status usage that is not supported for AMQP driver.